### ISSUE TITLE
[Question] Alternative with self-hosted database

### ISSUE THREAD

**Issue by amenk** [firebase/polymerfire#185]
I am searching a way to use a polymerfire based app in a LAN, without internet and firebase access.
Is there any alternatives which is polymer compatible and uses a firebase alternative?

Best would be a drop-in replacement the app side.

**Comment by tjmonsi** [comment]
I don't think that would be possible. Unless you know how firebase works on the backend or create a pseudo-firebase js to replace the original firebase which polymerfire uses, and then tweak all of the elements in polymerfire to use the pseudo-firebase js connector to your makeshift local database.

**Comment by justweb1** [comment]
Your best bet would probably to use pouchdb. you can find it at webcomponents.org.
